packages/front-end/components/Experiment/CompactResults.tsx

```diff
@@ -150,11 +185,42 @@ const CompactResults: FC<{
               );
             }
             return (
-              <tr key={m}>
-                <th>{metric.name}</th>
+              <tr
+                key={m}
+                className={`metricrow ${metric.inverse ? "inverse" : ""}`}
+              >
+                <th className="metricname">
+                  {metric.name}{" "}
+                  {metric.inverse ? (
+                    <div className="inverse-indicator tiptrigger">
```

Comment:
I think tooltips will be useful elsewhere. Can we make this it's own self-contained component?

```tsx
<Tooltip label={"This is my tooltip text"}>
  This is the trigger
</Tooltip>
```

Also, make sure the css rules for this are moved to the root level.